Implemented client side validation

sfcbetiuc opened the pull request:
- Removed cookies logic from ApiClient (change is made by updating
ApiClient.mustache)
- Regenerated the Models (changes are made by updating the
the partial_model_generic.mustache, api.mustache files)
- Added unit tests for the validation of the passed parameters of
the Campaign constructor
- Other minor refactorings in integration tests for CampaignApi, AssetApi

Commit:
Implemented client side validation

- Removed cookies logic from ApiClient (change is made by updating
ApiClient.mustache)
- Regenerated the Models (changes are made by updating the
the partial_model_generic.mustache, api.mustache files)
- Added unit tests for the validation of the passed parameters of
the Campaign constructor
- Other minor refactorings in integration tests for CampaignApi, AssetApi

sfdrogojan reviewed (approved):
Try to reduce the empty lines generated by swagger from the mustache files for validation.

Commit:
Regenerated models

- Models regenerated after updating the partial_model_generic.mustache file.
The changes in this mustache file were made in order to eliminate as many as
possible automatically generated new lines in the model classes.
- Refactored from authBasePath to authBaseUrl in autogenerated Api classes
